packages/@webex/plugin-meetings/src/members/index.ts

```diff
@@ -886,6 +886,31 @@ export default class Members extends StatelessWebexPlugin {
     });
   }
 
+  /**
+   * @param {String} memberId
+   * @param {boolean} [raise] - to raise hand (=true) or lower (=false), default: true
```

Comment:
JSDoc needs to be updated.

Reply:
Can refer to my PR: https://github.com/webex/webex-js-sdk/compare/next...stanjiawang:webex-js-sdk:SPARK-497506?expand=1

Reply:
```
  /**
   * Moves a meeting member into the lobby.
   *
   * @param  {string} memberId               - The ID of the member to move.
   * @returns {Promise<object>}              - Resolves with the lobby‐move response.
   * @public
   * @memberof Members
   */
   ```